### ISSUE TITLE
Masters problem

### ISSUE THREAD

**Issue by TwistedModding** [focustense/easymod#16]
Easynpc is adding some npc overhauls as a master (such as the bijin series), meaning i cant disable the plugins/mods without having a missing masters issue.

**Comment by focustense** [comment]
I tested quite a bit on those mods and didn't get that result, so I'll need more specifics. For which NPCs did it select those overhauls as default? Did this happen when you started a new profile, were they left over from an old profile, or did you use the Reset Defaults button from Maintenance?

In my own installation it successfully identifies all 3 Bijin mods as "overhaul mods", which means that no matter what else you have in your load order, it will _never_ choose them as the default plugin, it's blocked from doing so even if it can't find a better source than the master (Skyrim.esm).

Is it possible that you made these changes explicitly? Any mod you select as "default" for any NPC is going to become a master. If so, you only want to select _faces_ from overhauls, not defaults.

It might help to attach your autosave (profile.log) in %LocalAppData%\EasyNPC, assuming there's nothing in your profile that you'd rather keep private.

**Comment by focustense** [comment]
Related to this issue but not intended as a substitute for fixing it (if it turns out to be a bug), I've added #17 for making the process more transparent.

**Comment by focustense** [comment]
Since this hasn't been updated in a while, it's covered in the FAQ, and there's a separate issue for improving the UI, going to close.

**Comment by izomiac** [comment]
I encountered the same problem, with Bijin, Diversity, and Fluffjiits.  Those were the last three mods in the masters list (so no patches) and double clicking them showed zero NPC records using them as a default.  I was able to fix it by closing EasyNPC, opening %LOCALAPPDATA%\EasyNPC\Profile.log doing a search & replace for the offending plugins, saving the file, and reopening EasyNPC.  No clue why the GUI doesn't show these entries.  I used a simple regex search/replace to fix it:  

Search: 
`^({"master":")([^"]*)(".*DefaultPlugin.*)Diversity - A Character Overhaul.esp(.*)Diversity - A Character Overhaul.esp(.*)$`
Replace: 
`\1\2\3\2\4\2\5`
